docs/index.md

```diff
@@ -33,63 +33,63 @@ Other parts include a logging daemon, utilities to control basic system configur
 
 ---
 
-<pre style="color:white; background-color:black; font-size:smaller; padding:6pt 8pt">
-Welcome to <span style="color:blue">Fedora 20 (Heisenbug)</span>!
+<pre class="intro-code-block">
```

Comment:
I disagree on how this block looks in light mode. The brand green/blue makes for terrible contrast with the new lighter background color. Dark mode looks okay, though.